### PR TITLE
add manpage menu

### PR DESCRIPTION
**Pull request by kswiecicki**
This PR adds a manpage shortcut menu similar to the one we've had on the older gh-pages implementation.

---
This change is [Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/241)

### CODE REVIEW

**Review by lukaszstolarczuk — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/241)* status: 0 of 5 files reviewed, 2 unresolved discussions (waiting on @kswiecicki and @lukaszstolarczuk)

*[a discussion](https://reviewable.io/reviews/pmem/pmem.github.io/241#-N2Qh8N5EPOPW1IvYVdm:-N2Qh8N5EPOPW1IvYVdn:b-3huaap) (no related file):*
do you have a preview of that menu (e.g. on your fork)?

___
___
*[`data/libs_map.yml` line 1 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/241#-N2QhHVy06v19pkgvR_-:-N2QhHVy06v19pkgvR_0:b-qfo8qt) ([raw file](https://github.com/pmem/pmem.github.io/blob/250c21cdf072df4570586ee5e3fca847893cdd32/data/libs_map.yml#L1)):*
> ```YAML
> - libpmem: pmem_check_version.3 pmem_deep_drain.3 pmem_deep_flush.3 pmem_deep_persist.3 pmem_drain.3 pmem_errormsg.3 pmem_flush.3 pmem_has_auto_flush.3 pmem_has_hw_drain.3 pmem_is_pmem.3 pmem_map_file.3 pmem_memcpy.3 pmem_memcpy_nodrain.3 pmem_memcpy_persist.3 pmem_memmove.3 pmem_memmove_nodrain.3 pmem_memmove_persist.3 pmem_memset.3 pmem_memset_nodrain.3 pmem_memset_persist.3 pmem_msync.3 pmem_persist.3 pmem_unmap.3
> ```

this map seems like a fun thing to maintain :wink: can't this be somehow automatized? :wink:

**Comment by kswiecicki** [comment]
*[a discussion](https://reviewable.io/reviews/pmem/pmem.github.io/241#-N2Qh8N5EPOPW1IvYVdm:-N2QkZSa7pCrEo8c50se:bjwowv9) (no related file):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

do you have a preview of that menu (e.g. on your fork)?
</blockquote></details>

The preview on my fork is not displayed correctly. For some reason, CSS aren't applied on my fork.
I've even rebased on the main branch recently. I don't know what's the problem.

**Comment by kswiecicki** [comment]
*[a discussion](https://reviewable.io/reviews/pmem/pmem.github.io/241#-N2Qh8N5EPOPW1IvYVdm:-N2QrTdx7cwSnv_-p7rz:bvsaagl) (no related file):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

hmm ok, we may merge this and try "on the production" :wink: as long as you promise to fix that if broken :smile: 

nonetheless, it be nice to have any review from someone from Alfa Squad and we're good to go, I think
</blockquote></details>

You can try it locally using `hugo server` CLI command. I've shared this PR with Alfa Squad too.

File: data/releases_windows.yml [hunk 1]
```diff
@@ -0,0 +1,2665 @@
...
+        pmem_memmove_persist.3
+        pmem_memset.3
+        pmem_memset_nodrain.3
+        pmem_memset_persist.3
+        pmem_msync.3
+        pmem_persist.3
+        pmem_unmap.3
+
+    libpmem2:
+        pmem2_badblock_clear.3
```
Comment (wlemkows) [inline data/releases_windows.yml:51]: this is not up to date, what about new async man pages?

File: data/libs_map.yml [hunk 2]
```diff
@@ -0,0 +1,373 @@
+- libpmem: pmem_check_version.3 pmem_deep_drain.3 pmem_deep_flush.3 pmem_deep_persist.3 pmem_drain.3 pmem_errormsg.3 pmem_flush.3 pmem_has_auto_flush.3 pmem_has_hw_drain.3 pmem_is_pmem.3 pmem_map_file.3 pmem_memcpy.3 pmem_memcpy_nodrain.3 pmem_memcpy_persist.3 pmem_memmove.3 pmem_memmove_nodrain.3 pmem_memmove_persist.3 pmem_memset.3 pmem_memset_nodrain.3 pmem_memset_persist.3 pmem_msync.3 pmem_persist.3 pmem_unmap.3
```
Comment (wlemkows) [inline data/libs_map.yml:1]: We have to enable again/update scripts to autogenerate documentation on  this repo and auto genarate this yml (create  PR using pmdk bot account).
(I think changes on pmdk repo are needed)

**Review by wlemkows — CHANGES_REQUESTED** [review]

File: data/releases_linux.yml [hunk 3]
```diff
@@ -0,0 +1,2867 @@
...
+        pmemset_perror.3
+        pmemset_persist.3
+        pmemset_remove_part_map.3
+        pmemset_remove_range.3
+        pmemset_set_contiguous_part_coalescing.3
+        pmemset_source_delete.3
+        pmemset_source_from_file.3
+        pmemset_source_from_pmem2.3
+
+- tag: "1.10"
```
Comment (wlemkows) [inline data/releases_linux.yml:429]: 1.11 is missing?

**Comment by kswiecicki** [comment]
*[`data/libs_map.yml` line 1 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/241#-N2Qflkpai6acfLrxJRi-r2-1:-N2RcdeJ7PRGjT8bDMoY:b-6fyqph) ([raw file](https://github.com/pmem/pmem.github.io/blob/d6b16a925eaf07a7d898db1715117cbee3366704/data/libs_map.yml#L1)):*
<details><summary><i>Previously, wlemkows (Weronika Lewandowska) wrote…</i></summary><blockquote>

We have to enable again/update scripts to autogenerate documentation on  this repo and auto genarate this yml (create  PR using pmdk bot account).
(I think changes on pmdk repo are needed)
</blockquote></details>

I've done that and updated the script: https://github.com/pmem/pmdk/pull/5451.
I'm not sure if the same pmem-bot github token will be valid on pmem/pmem.github.io repo.

**Review by kswiecicki — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/241)* status: 0 of 5 files reviewed, 5 unresolved discussions (waiting on @lukaszstolarczuk and @wlemkows)

___
*[`data/libs_map.yml` line 1 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/241#-N2QhHVy06v19pkgvR_-:-N2RcW2bD76x5qu0vwYP:be4dzqy) ([raw file](https://github.com/pmem/pmem.github.io/blob/250c21cdf072df4570586ee5e3fca847893cdd32/data/libs_map.yml#L1)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

this map seems like a fun thing to maintain :wink: can't this be somehow automatized? :wink: 
</blockquote></details>

I've re-enabled auto doc generation https://github.com/pmem/pmdk/pull/5451.
___
*[`data/releases_linux.yml` line 429 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/241#-N2Qflkpai6acfLrxJRj-r2-429:-N2Rcxl0Br-4Blv-vyew:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/d6b16a925eaf07a7d898db1715117cbee3366704/data/releases_linux.yml#L429)):*
<details><summary><i>Previously, wlemkows (Weronika Lewandowska) wrote…</i></summary><blockquote>

1.11 is missing?
</blockquote></details>

Done.
___
*[`data/releases_windows.yml` line 51 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/241#-N2Qflkpai6acfLrxJRk-r2-51:-N2Rcyi4C9dcqmvUd20p:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/d6b16a925eaf07a7d898db1715117cbee3366704/data/releases_windows.yml#L51)):*
<details><summary><i>Previously, wlemkows (Weronika Lewandowska) wrote…</i></summary><blockquote>

this is not up to date, what about new async man pages?
</blockquote></details>

Done.

**Review by lukaszstolarczuk — COMMENTED** [review]
> Reviewed 1 of 5 files at r1.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/241)* status: 1 of 5 files reviewed, 4 unresolved discussions (waiting on @kswiecicki, @lukaszstolarczuk, and @wlemkows)

___
*[`data/libs_map.yml` line 1 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/241#-N2QhHVy06v19pkgvR_-:-N2VSkNK9Ykr9ZCY05qP:b-q74txj) ([raw file](https://github.com/pmem/pmem.github.io/blob/250c21cdf072df4570586ee5e3fca847893cdd32/data/libs_map.yml#L1)):*
<details><summary><i>Previously, kswiecicki (Krzysztof Święcicki) wrote…</i></summary><blockquote>

I've re-enabled auto doc generation https://github.com/pmem/pmdk/pull/5451.
</blockquote></details>

great!
___
*[`data/releases_linux.yml` line 527 at r3](https://reviewable.io/reviews/pmem/pmem.github.io/241#-N2VTzru6n1KKsg5tL5d:-N2VTzru6n1KKsg5tL5e:b-5dir54) ([raw file](https://github.com/pmem/pmem.github.io/blob/c1fb125fdf7e28102520039a31b5c21576cbbf63/data/releases_linux.yml#L527)):*
> ```YAML
>         pmem2_map_get_store_granularity.3
>         pmem2_map_new.3
>         pmem2_memcpy_async.3
> ```

async in 1.11? did you mean to paste it into Windows yml?

**Review by lukaszstolarczuk — COMMENTED** [review]
> Reviewed 1 of 5 files at r1, 1 of 1 files at r2, 2 of 2 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/241)* status: all files reviewed, 3 unresolved discussions (waiting on @wlemkows)

___
*[`data/releases_linux.yml` line 527 at r3](https://reviewable.io/reviews/pmem/pmem.github.io/241#-N2VTzru6n1KKsg5tL5d:-N2Vr3Qt7gm9X2b4Ys1J:bsouy8w) ([raw file](https://github.com/pmem/pmem.github.io/blob/c1fb125fdf7e28102520039a31b5c21576cbbf63/data/releases_linux.yml#L527)):*
<details><summary><i>Previously, kswiecicki (Krzysztof Święcicki) wrote…</i></summary><blockquote>

My bad, I've messed up the release tags.
Now I've updated `releases_linux/windows.yml` with content from `libpmem2.link.in libpmemset.link.in` and `libpmem2.def libpmemset.def` files from the `stable-1.11` and `master` branches.
</blockquote></details>

sounds alright :+1:

**Review by wlemkows — COMMENTED** [review]
:lgtm:

> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/241#-:-N2jkdu-Dqi5xktTkStt:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @kswiecicki)

**Pull request merged**